src/test/java/io/openliberty/tools/intellij/it/Utils/ItConstants.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.tools.intellij.it.Utils;
+
+public class ItConstants {
+
+    /**
+     * Constants for Integration testing
+     */
+    public static final String SYSTEM_RESOURCE_JAVA = "SystemResource.java";
+    public static final String SYSTEM_RESOURCE_2_JAVA = "SystemResource2.java";
+    public static final String SYSTEM_RESOURCE = "SystemResource";
+    public static final String SYSTEM_RESOURCE_2 = "SystemResource2";
+    public static final String SERVICE_LIVEHEALTH_CHECK = "ServiceLiveHealthCheck";
+
+    public static final String[] CONFIG_DIR_PATH = { "src", "main", "liberty", "config" };
+    public static final String[] SYSTEM_DIR_PATH = { "src", "main", "java", "io.openliberty.mp.sample", "system"};
+    public static final String[] HEALTH_DIR_PATH = { "src", "main", "java", "io.openliberty.mp.sample", "health" };
```

Comment:
Why use dots as part of the String values in the String arrays `SYSTEM_DIR_PATH` and `HEALTH_DIR_PATH`? The original hard coded values do not use any dots in most cases. Now you have to convert all the dots to slashes. Why not just remove the dots from the constants so you do not have to do that conversion each time?

Note, there are some spots - like https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/src/test/java/io/openliberty/tools/intellij/it/SingleModJakartaLSTestCommon.java#L201 - where dots have to be used because of the way the UI robot expects the values. For these cases, we can use a different constant.